docs/csa/data-lakes.md

```diff
@@ -0,0 +1,95 @@
+
+If you’re not sure where to start, consider asking these questions:
+* What kind of data sources can you ingest from?
+* How is the data searchable?
+* How much storage capacity do you need?
+* What is the event-per-second throughput? 
+
+#### Step 3: Implement data ingestion
+
+Set up a process for collecting data into your data lake. With Sumo Logic, you can begin data ingestion in a secure and compliant manner out-of-the-box. Our high event-per-second throughput ensures you can handle the most demanding workloads and data ingestion without issue.
+
+#### Step 4: Enforce data security
+
+Security controls are necessary to protect your data lake from unauthorized access and breaches. At Sumo Logic, we use state-of-the-art encryption techniques to safeguard your data in transit and at rest. But it’s not just about protection. You must also prioritize compliance and monitoring to ensure your data lake meets all regulatory requirements. Sumo Logic holds numerous certifications and attestations. 
+
+#### Step 5: Search your security data
+
+With the vast amounts of data generated by security tools, it can be overwhelming to sift through it all. Sumo Logic's [Search Query Language](/docs/search/search-query-language/) helps identify anomalous behavior and flags potential security threats in real-time. Query your data lake to uncover patterns, investigate alerts, and identify potential security risks. 
+
+#### Step 6: Establish data governance
+
+Sumo Logic uses role-based access control (RBAC), data classification, retention, and archiving. Our data classification tools allow you to easily identify and manage sensitive information, ensuring that only authorized users have access. And you can keep your data for as long as you need to meet regulatory requirements while also ensuring that it is easily accessible.
+
+#### Step 7: Analyze security data
+
+Once you connect your data lake to Sumo Logic, you can start analyzing security data. Cloud Security Analytics is built on a security-first principle. It is designed to analyze security data with real-time threat detection, compliance reporting, and dashboards. Our cloud-native architecture unifies security events and investigations across multiple cloud platforms, including AWS, Azure, and Google Cloud Platform.
+
+#### Step 8: Monitor your data
+
+With security data flowing in from your data lake, you can continuously monitor and analyze your data to identify potential threats and respond proactively. Our platform is also updated with the latest security content from our Sumo Logic Threat Labs team. This team moves quickly to support customers in crisis, like when the Log4j vulnerability surfaced. They continuously monitor and analyze data to identify threats, both known and unknown.
+
+## Install a data lake for use with Cloud Security Analytics
```

Comment:
It seems like this should be higher up in the doc

Reply:
Good point. For now, we'll leave it as-is.